tests/integration/suite/scheduler/quorum/notls.go

```diff
@@ -95,13 +106,19 @@ func (n *notls) Run(t *testing.T, ctx context.Context) {
 
 	client := schedulerv1pb.NewSchedulerClient(conn)
 
+	appID := n.daprd.AppID()
+
 	req := &schedulerv1pb.ScheduleJobRequest{
 		Job: &runtimev1pb.Job{
 			Name:     n.jobName,
-			Schedule: "@every 1s",
+			Schedule: "@every 2s",
+			Data: &anypb.Any{
+				TypeUrl: "type.googleapis.com/google.type.Expr",
+			},
 		},
 		Metadata: map[string]string{
 			"namespace": "default",
+			"appID":     appID,
```

Comment:
I added the namespace and appID as first class fields, BUT for now, it will remain in the metadata as well until the cron library sends back those fields, bc currently it doesn't and I still need a way to lookup the ns + appID for the streaming back to the sidecar from the scheduler and that is only done by the lookup of the metadata. I added TODOs to rm it once the library accts for sending it back separately from the metadata

Reply:
The cron library is not aware of namespace or appID concepts. It just knows jobName, data and metadata.

Reply:
We can change the scheduler to get those attributes and add them to the metadata, so those get persisted.

Reply:
I've made the `metadata` -> `errMetadata` to add the appID/ns to the err but not to be used as the metadata to be sent to scheduler, the err standardization should be updated with the new format, but not in this PR.

I've added a todo to add metadata to the get/delete/schedule funcs in the universal dir once the state store logic is added, but removed it for now and add the metadata in the scheduler for the schedule func